### PR TITLE
Juanro/fix employees table

### PR DESCRIPTION
**Pull request by Juanro-Minne**
Optimized the employees table to use a single datasource and added parameter to the api call

image
image
image
image

### CODE REVIEW

**Review by ReinoRabbit — APPROVED** [review]

File: src/app/services/hris/employee/employee.service.ts [hunk 1]
```diff
@@ -79,9 +79,12 @@ export class EmployeeService {
   *
   * @returns List of EmployeeDto objects.
   */
-  filterEmployees(championID: number, employeeType: number): Observable<EmployeeProfile[]> {
-    var queryParams = `?PeopleChampId=${encodeURIComponent(championID)}`
-    queryParams += `&employeeType=${encodeURIComponent(employeeType)}`
-    return this.httpClient.get<EmployeeProfile[]>(`${this.baseUrl}/filter-employees${queryParams}`);
-  }
+ // employee.service.ts
```
Comment (MatthewSchoeman) [inline src/app/services/hris/employee/employee.service.ts:82]: Is this comment necessary

**Review by MathildaShongwe15 — APPROVED** [review]

File: src/app/services/hris/employee/employee.service.ts
Comment (MatthewSchoeman) [inline src/app/services/hris/employee/employee.service.ts:1]: Please can summaries be added to the rest of the methods

File: src/app/services/hris/employee/employee.service.ts (hunk 1 above)
Comment (Juanro-Minne) [reply src/app/services/hris/employee/employee.service.ts]: removed

**Review by MathildaShongwe15 — COMMENTED** [review]
I noticed that the "items per page"  at the bottom does not update when you filter, it's still stuck on 10 items per page.

File: src/app/services/hris/employee/employee.service.ts
Comment (Juanro-Minne) [reply src/app/services/hris/employee/employee.service.ts:1]: summaries added

**Comment by Juanro-Minne** [comment]
> I noticed that the "items per page" at the bottom does not update when you filter, it's still stuck on 10 items per page.

as far as i know thats the intervals of records to display so that wouldnt change

**Review by ReinoRabbit — APPROVED** [review]

**Review by tbddorothy — APPROVED** [review]

**Review by MathildaShongwe15 — APPROVED** [review]

**Pull request merged**